packages/host/src/LanguageMenu.tsx

```diff
@@ -7,20 +7,23 @@ import {
   useHostContext,
 } from '@openmsupply-client/common';
 import { SupportedLocales } from '@openmsupply-client/common/src/intl/intlHelpers';
+import { useNavigate } from 'react-router';
 
 interface LanguageMenuItemProps {
   children: string;
   language: SupportedLocales;
 }
 
 export const LanguageMenu: React.FC = () => {
+  const navigate = useNavigate();
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
   const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
     setAnchorEl(event.currentTarget);
   };
 
   const handleClose = () => {
     setAnchorEl(null);
+    navigate(0);
```

Comment:
required for the RTL / LTR swapping?

Reply:
Yeah, for a couple of the things acting super weird when switching to rtl at runtime that I spent way to long trying to fix. This is a hack that I landed on. Can take it away and check the styles which aren't switched.. just a couple and super annoying 😭

Reply:
while it is neater without the reload - in practice I'm not expecting people to regularly switch language and the reload is pretty quick (at this stage) anyway. so it's all good - still quicker than editing a user, logging out and logging in again :wink: